exporter/kafkaexporter/README.md

```diff
@@ -61,6 +61,13 @@ The following settings can be optionally configured:
   User should calculate this as `num_seconds * requests_per_second` where:
     - `num_seconds` is the number of seconds to buffer in case of a backend outage
     - `requests_per_second` is the average number of requests per seconds.
+- `compression` (default = none): The compression codec to compress the messages sent to Kafka. Four compression methods are supported: `gzip`, `snappy`, `lz4` and `zstd`.
+- `compression_level` (default = -1000): The corresponding compression level which measures the compression degree.
+  Lower compression level would lead to fast compression as well as relatively big file size. The valid ranges for each compression method are:
+  - `gzip`: `1 - 9`
+  - `snappy`: Snappy does not support compression level.
+  - `lz4`: `1 - 17`
+  - `zstd`: `1 - 22`
```

Comment:
I would rewrite this config like the one for `auth`, i.e., something like
```
kafka:
  compression:
    gzip:
      level: 5
```

and allow the user to fill in only one section inside this one.

That way we can extend the configuration more easily in the future and it is more consistent with existing configuration options.

Reply:
This structure makes more sense. I will rewrite this.